client/app/components/composites/MenuMobile/MenuMobile.css

```diff
@@ -0,0 +1,123 @@
+.menumobile {
+
+  &:focus,
+  & :focus {
+    outline: none;
+    color: var(--Menu_textColorFocus);
+  }
+
+  &:focus .menuLabelMobile,
+  &.canvasOpen .menuLabelMobile {
+    color: var(--Menu_textColorFocus);
+  }
+
+  &:focus .menuLabelMobileIcon g,
+  &.canvasOpen .menuLabelMobileIcon g {
+    fill: var(--Menu_textColorFocus);
+  }
+}
+
+.menuLabelMobile {
+  display: inline-block;
+  padding: var(--Menu_labelPaddingVertical) var(--Menu_labelPaddingHorizontal);
+  cursor: pointer;
+}
+
+
+.overlay {
+  display: none;
+  cursor: pointer;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 9;
+  height: 100vh;
+  width: 100vw;
```

Comment:
Wouldn't 100% width and height work here just as well, and have better browser support?